doc/source/whatsnew/v0.25.0.rst

```diff
@@ -741,6 +741,44 @@ consistent with NumPy and the rest of pandas (:issue:`21801`).
    cat.argsort()
    cat[cat.argsort()]
 
+.. _whatsnew_0250.api_breaking.list_of_dict:
+
+Column order is preserved when passing a list of dicts to DataFrame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Starting with Python 3.7 the key-order of `dict` is `guaranteed <https://mail.python.org/pipermail/python-dev/2017-December/151283.html>`_. In practice, this has been true since
+Python 3.6. The DataFrame constructor now treats a list of dicts in the same way as
+it has a list of `OrderedDict` since v0.19.0. This change applies only when pandas is running
+on python>=3.6 (:issue:`27309`). As a consequence, the column order produced by `DataFrame()`
+in such cases has changed.
+
+.. code-block:: ipython
+
+   In [1]: data = [
+      ...:     {'name': 'Joe', 'state': 'NY', 'age': 18},
+      ...:     {'name': 'Jane', 'state': 'KY', 'age': 19}
+      ...: ]
+
+*Previous behavior* (lexicographically sorted)
+
+.. code-block:: ipython
+
+   In [1]: pd.DataFrame(data)
+   Out[1]:
+      age  name state
+   0   18   Joe    NY
+   1   19  Jane    KY
+
+*New behavior*: preserve order of the dicts
+
+.. code-block:: ipython
```

Comment:
don't need the prompts; ipython adds these

Reply:
This is what all other segments in the file do.